chainer/training/extensions/plot_report.py

```diff
@@ -50,6 +47,15 @@ class PlotReport(extension.Extension):
     It also adds ``'epoch'`` and ``'iteration'`` entries to each result
     dictionary, which are the epoch and iteration counts at the output.
 
+    If your environment needs to specify a backend of matplotlib explicitly,
+    please call ``matplotlib.use`` before importing Chainer. For example,
+
+    .. code-block:: python
+        import matplotlib
+        matplotlib.use('Agg')
+
+        import chainer
```

Comment:
Please also write that `matplotlib.use` is disabled once the user imports `PlotReport`.